### ISSUE TITLE
能不能给个正儿八经的模型地址,原来的跑路了,其他的试了若干次也不行

### ISSUE THREAD

**Issue by Wz-2024** [cure-lab/MMA-Diffusion#21]

**Comment by RYC-98** [comment]
初步尝试，下面这个可以载入成功，https://www.modelscope.cn/models/AI-ModelScope/stable-diffusion-inpainting/summary
![image](https://github.com/user-attachments/assets/fcc9c411-3d38-4ecb-b3f6-d0e6ae445c0e)

**Comment by Wz-2024** [comment]
您跑通这个项目的第二个方法了吗

**Comment by RYC-98** [comment]
imageattack可以跑的。之前报错的原因貌似是导入的pt文件torchvision版本太低了，因此用下面命令重新生成一个pt文件，然后导入即可：
import torch
import torchvision
from torchvision.transforms import Compose, RandomResizedCrop, ToTensor, Normalize

processor = Compose([
RandomResizedCrop(size=(224, 224), scale=(0.9, 1.0), ratio=(0.75, 1.3333333333333333), interpolation=torchvision.transforms.InterpolationMode.BICUBIC),
ToTensor(),
Normalize(mean=(0.48145466, 0.4578275, 0.40821073), std=(0.26862954, 0.26130258, 0.27577711))
])

torch.save(processor, "./new_open_clip_preprocess.pt")

processor = torch.load("./new_open_clip_preprocess.pt")
print(processor)

**Comment by Wz-2024** [comment]
好的，多谢
